web/tests/lib/events.js

```diff
@@ -151,6 +152,7 @@ exports.fixtures = {
                 order: 2,
                 display_in_profile_summary: false,
                 required: false,
+                editable_by_user: false,
```

Comment:
These changes should be in the first commit adding the field in backend, so that `tools/check-schemas` test pass for each commit.

Reply:
Moved this change to first commit.